lib/msal-angular/docs/v2-docs/events.md

```diff
@@ -56,7 +56,7 @@ ngOnInit(): void {
 }
 ```
 
-For the full example of using events, please see our sample [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/d46c4455243bd51767f669a13fbb717d786c6716/samples/msal-angular-v2-samples/angular11-sample-app/src/app/home/home.component.ts#L17).
+For the full example of using events, please see our sample [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/samples/msal-angular-v3-samples/angular15-sample-app/src/app/home/home.component.ts).
```

Comment:
This is in the v2 docs folder but pointing to a v3 sample?

Reply:
Good point. I ended up moving all docs out of the v2 folder instead.